python/tskit/trees.py

```diff
@@ -5935,6 +5935,111 @@ def trim(self, record_provenance=True):
         tables.trim(record_provenance)
         return tables.tree_sequence()
 
+    def decapitate(self, time, record_provenance=True):
+        """
+        Return a copy of this tree sequence with topology and mutation information
+        older than the specified time removed. Please see the
+        :meth:`.TableCollection.decapitate` method for details.
+
+        :param float time: The cutoff time.
+        :param bool record_provenance: If True, add details of this operation to the
+            provenance information of the returned tree sequence. (Default: True).
+        :rtype: tskit.TreeSequence
+        """
+        tables = self.dump_tables()
+        tables.decapitate(time, record_provenance=record_provenance)
+        return tables.tree_sequence()
+
+    def split_edges(self, time, *, flags=None, population=None, metadata=None):
+        """
+        Returns a copy of this tree sequence in which we replace any
+        edge ``(left, right, parent, child)`` in which
+        ``node_time[child] < time < node_time[parent]`` with two edges
+        ``(left, right, parent, u)`` and ``(left, right, u, child)``,
+        where ``u`` is a newly added node for each intersecting edge. If
+        ``metadata`` or ``population`` are specified, newly added nodes will be
+        assigned these values. Otherwise, default values will be used. If a
+        metadata schema is defined for the node table, the empty dictionary
+        will be used by default for the new node when calling
+        :meth:`.NodeTable.add_row`; otherwise, an empty byte string.
```

Comment:
```suggestion
        assigned these values. Otherwise, default values will be used.
        The default metadata is an empty dictionary if a
        metadata schema is defined for the node table,
        and is an empty byte string otherwise.
```